docs/source/data/fetching-data.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Fetching Data
+description: Fetching data from data sources besides REST
+---
+
+<!--  (TODO) this whole article! -->
```

Comment:
TODO(AS4) if we want to catch before AS4 entirely done
FIXME if you don't want to merge